pegasus/sites.v3/code.org/public/educate/resources/videos.md

```diff
@@ -10,6 +10,62 @@ style_min: true
 
 We are building a growing library of educational videos available for re-use by educators worldwide, online or in classrooms. Our goal is to enable our efforts to be amplified beyond our own curriculum's reach. **Note:** see below for the licensing restrictions on use of these videos.
 
+## How Computers Work
+
+<div class="col-50" style="float: left; padding: 10px;">
+
+<% facebook = {:u=>'https://youtu.be/OAx_6-wdslM'} %>
+<% twitter = {:url=>'https://youtu.be/OAx_6-wdslM', :related=>'codeorg', :text=>'Introducing How Computers Work @codeorg'} %>
+<%=view :display_video_thumbnail, id: "how_computers_work", video_code: "OAx_6-wdslM", caption: "Introducing How Computers Work", play_button: 'center', facebook: facebook, twitter: twitter, download_path: "//videos.code.org/cs-discoveries/concept/01_HowComputersWork_sm.mp4" %>
```

Comment:
Is `01_HowComputersWork_sm.mp4` the best filename?  Users will see this if they download the video.  Seems like we could do something like `CodeOrg_HowComputersWork_01.mp4`.

Reply:
I agree. I didn't choose it. Someone already uploaded it like that so if I'll break the link if I rename. I could rename where we use it on the site, but it may already be used externally in emails or resources.